lib/web_ui/dev/test_runner.dart

```diff
@@ -582,3 +647,17 @@ void _copyTestFontsIntoWebUi() {
     sourceTtf.copySync(destinationTtfPath);
   }
 }
+
+/// This objest is used as an input message to the PoolResources that are
+/// building the tests.
+class TestBuildInput {
+  /// Target to build.
```

Comment:
The terms "test", "input", and "target" all mean the same thing in this file. I think it would improve clarity if we used one term. I think I would prefer "test". For example, this class can be named `Test` or `EngineTest`, and the `target` field can simply be `path`.

Reply:
done